src/libs/Navigation/linkingConfig.js

```diff
@@ -0,0 +1,65 @@
+export default {
```

Comment:
Just have a few suggestions around deeplinks in general:
- I would suggest creating auth & unauth(public) stacks and prefix the urls with one of those values. For example the path would be `expensify-cash://Unauthorized/SignIn` for a public route and `expensify-cash://Authorized/Home`.
- What this allows you to do is in the root nav where you have your linking logic, you can see if the route begins with Authorized and if there's no authToken then you fallback to `SignIn`. Similarly, if there is an auth token, you might not want them to go to an unauth route and fallback to home instead.
- might want to include `https://www.expensify.cash` to the list of prefixes as well.
- I think i saw some custom logic around adding Home as the root screen since you want that to always be the initiate screen. another way to achieve this is to just do something like `expensify-cash://Authorized/Home?navigateTo=Settings`. Basically this would take you to home then you can wait for all your Api requests to resolve before navigating them to the desired path (Settings)
- Unless you're versioning deeplinks, on mobile you can have users on older versions that click a deeplink for a screen that doesn't exist. You can redirect them to a page that stores the deeplink and suggests them to update, then once they update you can try navigating again.

Reply:
That's an interesting idea to have the auth and unauth routes. I might not totally be understanding this point, but am I correct in thinking that the urls on web would always be prefixed with `auth` or `unauth` something like `http://expensify.cash/auth/r/123`?

I like the other ideas. I'm not too sure exactly what it would take to apply them or if we need all of it yet, but definitely open to more detailed proposals.

Reply:
ah actually most of my suggestions were around specifically mobile deeplinks since most of my experience has been specifically for mobile, but looks like these are also used to define the web routes as well